fecgov/marshmallow-pagination
# update marshmallow-sqlalchemy req and bump marshmallow-paginiation ve…

### Summary
Resolves issue #2:
- updates `marshmallow-sqlalchemy` requirement to be >= rather than == (helps with openFEC requirements pinning)
- bumps version to 0.1.2
- removes unnecessary .pyc files

### How to test
from openFEC project directory:
1. pip install -r requirements.txt
2. pip install -r requirements-dev.txt
3. pip install git+https://github.com/fecgov/marshmallow-pagination@feature/2-update-marshmallow-sqlalchemy-dependency
4. pytest (verify tests pass)
5. pip install pipdeptree then run `pipdeptree` and scroll to top of output and verify no "Warnning!!!" exists for marshmallow-sqlalchemy or marshmallow-pagination
6. (optional) pip uninstall pipdeptree
